Assets/Shopify/UIToolkit/Components/RemoteImageLoader.cs

```diff
@@ -80,33 +80,38 @@ public class RemoteImageLoader : MonoBehaviour {
             );
         }
 
-        private void LoadImageURL(string url, RemoteImageCompletionDelegate completion) {
+        private void LoadImageURL(string url, RemoteImageCompletionDelegate onImage) {
             if (UseCache) {
-                StartCoroutine(CachedLoadImageURLRoutine(url, completion));
+                StartCoroutine(CachedLoadImageURLRoutine(url, onImage));
             } else {
-                StartCoroutine(LoadImageURLRoutine(url, completion));
+                StartCoroutine(LoadImageURLRoutine(url, onImage));
             }
         }
 
-        private IEnumerator LoadImageURLRoutine(string url, RemoteImageCompletionDelegate completion) {
+        private IEnumerator LoadImageURLRoutine(string url, RemoteImageCompletionDelegate onImage) {
```

Comment:
Nit: I would also rename the delegate to like `OnRemoteImageDelegate` or something